### PR TITLE
Use threads with configurable stack sizes in the content handler.

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: content_handler/content_handler_thread.cc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  pthread_attr_destroy(&thread_attributes);
+
+  return result == 0;
+}
+
+Thread::Thread()
+    : task_runner_(ftl::MakeRefCounted<mtl::internal::IncomingTaskQueue>()) {
+  valid_ =
+      CreateThread(&thread_, [](Thread* thread) { thread->Main(); }, this, 1e6);
```
Comment (jason-simmons) [inline content_handler/content_handler_thread.cc]: maybe use a default size of 1MB (1 << 20), which will likely be an exact multiple of the page size?

**Comment by jason-simmons** [comment]
LGTM

**Comment by chinmaygarde** [comment]
Done. Regenerating licenses.

**Comment by chinmaygarde** [comment]
@abarth I feel a bit weird about using `IncomingTaskQueue` from the `internal` namespace. I am not sure why it is marked as such. Do you want me to write our own?

**Comment by abarth** [comment]
> I am not sure why it is marked as such.

I didn't consider the use case of wanting to start a thread using a different API.

> Do you want me to write our own?

I'd prefer to just move the class outside of the internal namespace.

**Comment by chinmaygarde** [comment]
> I'd prefer to just move the class outside of the internal namespace.

I'll do this in a separate commit.

**Pull request merged**